extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -21,11 +29,17 @@
      * <li>does not declare any producer which is eligible for injection to any injection point,</li>
      * <li>is not directly eligible for injection into any {@link javax.enterprise.inject.Instance} injection point</li>
      * </ul>
+     *
+     * If set to none (or false) no beans will ever be removed even if they are unused (according to the criteria
+     * set out above)
+     *
+     * If set to fwk, then all unused beans will be removed, except the unused beans whose classes are declared
+     * in the application code
      * 
      * @see UnremovableBeanBuildItem
      */
-    @ConfigItem(defaultValue = "true")
-    public boolean removeUnusedBeans;
+    @ConfigItem(defaultValue = "all")
+    public String removeUnusedBeans;
```

Comment:
Should `framework` be the default? I think it's by far the most user friendly value.

Reply:
Actually I think that all should the default so we can have maximum "DCE" and `framework` should only be used if necessary.

Reply:
I'd keep `all`. I'm aware of the fact that it's not "the most user friendly value" but it's 1. well documented, 2. IMHO fits into quarkus opt"approach" (or at least ), 3. now we also have `@Unremovable`.